instructor/common.py

```diff
@@ -482,10 +505,13 @@ def strain_treatments_edit(request):
                               context_instance=RequestContext(request))
 
 
-def create_treatments(assignment):
-    drugs = models.Drug.objects.filter(assignment=assignment)
-    temperature = models.Temperature.objects.filter(assignment=assignment)
-    collection_times = models.CollectionTime.objects.filter(assignment=assignment)
+def create_treatments(assignment, drugs=None, temperature=None, collection_times=None):
```

Comment:
Can you rename `temperature` to `temperatures` for consistency with the other arguments?